src/kvstore/kvstore_base.c

```diff
@@ -104,12 +105,33 @@ int kvs_fid_from_str(const char *fid_str, kvs_idx_fid_t *out_fid)
 	return cortx_kvs_fid_from_str(fid_str, out_fid);
 }
 
-int kvs_alloc(struct kvstore *kvstore, void **ptr, size_t size)
+static int __kvs_alloc(struct kvstore *kvstore, void **ptr, size_t size)
 {
 	dassert(kvstore);
 	return kvstore->kvstore_ops->alloc(ptr, size);
 }
 
+int kvs_alloc(struct kvstore *kvstore, void **ptr, size_t size)
+{
+	size_t rc;
+	uint64_t myopid = perf_id_gen();
+	struct perfc_tls_ctx ptc = {.opid = myopid, .fn_tag = PFT_KVS_ALLOC};
+
+	dassert(kvstore);
+	perfc_tls_push(myopid, PFT_KVS_ALLOC);
```

Comment:
All ADDB related calls should be under ifdef .. isn't it ? Something like 
#ifdef ENABLE_TSDB_ADDB
     < all code related to ADDB >
#endif 
      <actual function>
...
<other ADDB related calls in similar manner >
...

Reply:
Sure let me address this, this support is added inside the performance counter macros, but adding here in the callers will also be good.

Reply:
Addressed this review comment in all the PRs of this story, please check.